lib/waistband/index.rb

```diff
@@ -29,6 +29,10 @@ def initialize(index_name, options = {})
         @subs = @subs.flatten     if @subs.is_a?(Array)
       end
 
+      if options['force_name'].present?
+        @force_name = options['force_name']
```

Comment:
Is there a reason to call this `force_name` versus `alias_name`?

Reply:
Because an index name could also be used.